lib/wal/tests/CallbackTest.cpp

```diff
@@ -0,0 +1,41 @@
+//
+// Created by Zoe Roux on 5/21/21.
+//
+
+#include <catch2/catch.hpp>
+#include <stdexcept>
+#include <Entity/Entity.hpp>
+#include "Models/Callback.hpp"
+
+using namespace WAL;
+
+TEST_CASE("Callback basic test", "[Callback]")
+{
+	Callback<std::string> callback;
+
+	REQUIRE_NOTHROW(callback("1"));
+	SECTION("Check add") {
+		int id =callback.addCallback([](const std::string& i) {
```

Comment:
where's the space ? !
```suggestion
		int id = callback.addCallback([](const std::string& i) {
```